projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.spec.ts

```diff
@@ -1,25 +1,70 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { GoPanelComponent } from './go-panel.component';
+import { GoActionSheetComponent } from '../go-action-sheet.component';
+import { ElementRef } from '@angular/core';
+
+class MockGoActionSheetComponent extends GoActionSheetComponent {
+  showContent: boolean = false;
+}
+
+// Need this to override an ElementRef injection param
+// that is unnecessary for this test
+// tslint:disable-next-line: max-classes-per-file
+class MockElementRef { }
 
 describe('GoPanelComponent', () => {
   let component: GoPanelComponent;
   let fixture: ComponentFixture<GoPanelComponent>;
+  let parentComponent: GoActionSheetComponent;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ GoPanelComponent ]
+      declarations: [ GoPanelComponent ],
+      providers: [
+        { provide: ElementRef, useClass: MockElementRef },
```

Comment:
If you change this to `{ provide: ElementRef, useValue: new ElementRef(MockGoActionSheetComponent) },` you don't need the MockElementRef class, but I'll leave this up to you as I'm not sure which one is better

Reply:
I like not having the extra class if we don't need to, thanks for this suggestion. I switched it.